### PR TITLE
feat: kickstart landing page revamp

### PR DESCRIPTION
**Pull request by eragon512**
**What changed? Why?**

**Notes to reviewers**

**How has it been tested?**
Manual

||Light|Dark|
|---|---|---|
|Home page|Screenshot 2024-04-20 at 3 24 20 AM|Screenshot 2024-04-20 at 3 34 00 AM|
|Docs page|Screenshot 2024-04-20 at 3 24 31 AM|Screenshot 2024-04-20 at 3 34 11 AM|

### CODE REVIEW

File: site/vocs.config.tsx [hunk 1]
```diff
@@ -66,40 +65,27 @@ export default defineConfig({
...
+          light: '#373A3E',
+          dark: '#FFFFFF',
+        }
+      }
+    }
   },
   topNav: [
+    { text: 'Examples', link: '/examples' },
     { text: 'Docs', link: '/getting-started', match: '/docs' },
-    {
```
Comment (Zizzamia) [inline site/vocs.config.tsx:83]: Let's keep the navigation as it is...

We can update it maybe later, but for now those links are all super useful.

Comment (eragon512) [reply site/vocs.config.tsx]: reset the topNav to the original state

File: site/docs/pages/index.mdx [hunk 2]
```diff
@@ -4,30 +4,20 @@ description: React components and TypeScript utilities for top-tier onchain apps
...
-        </div>
-        <div className="flex justify-center space-x-2">
-          <HomePage.Button href="/getting-started" variant="accent">Get started</HomePage.Button>
-          <HomePage.Button href="https://github.com/coinbase/onchainkit">GitHub</HomePage.Button>
-        </div>
+  <div className="mx-auto max-w-xl">
+    <div className="md:text-center flex flex-col gap-8 pt-48 pb-16">
+      <h1 className="text-center text-8xl font-medium no-underline">OnchainKit</h1>
+      <div className="text-center text-3xl leading-none">
+        <p className="leading-none">React components and TypeScript utilities to help you build top-tier onchain apps.</p>
```
Comment (Zizzamia) [inline site/docs/pages/index.mdx:16]: you probably want to keep the dark energy here
```
<span className="text-black dark:text-white">
```
for components and utilities in the dark theme.

Comment (eragon512) [reply site/docs/pages/index.mdx:16]: I've addressed this at the theme config directly: https://github.com/coinbase/onchainkit/pull/296/files#diff-43462e5b36aa73561e9f81ebc5ae883c35c8f73e60257d7a0f9aacaa9a34e597R75-R79

felt more appropriate to have consisten theme-level colors for text

**Review by Zizzamia — APPROVED** [review]

**Pull request merged**